addons/rook-ceph/enable

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python3
+
+import os
+import pathlib
+import subprocess
+
+import click
+
+SNAP = pathlib.Path(os.getenv("SNAP") or "/snap/microk8s/current")
+KUBECTL = SNAP / "microk8s-kubectl.wrapper"
+
+
+def clone_repository(repo_url, destination_dir, branch):
+    """
+    Clone a particular branch of a git repository to a destination directory.
+
+    :param repo_url: The repository URL.
+    :param destination_dir: The destination directory.
+    :param branch: The branch to clone.
+    """
+    click.echo(f"Cloning repository version {branch} in {destination_dir}")
+    try:
+        subprocess.call("git clone {} {} -b {}".format(repo_url, destination_dir, branch).split())
+    except subprocess.CalledProcessError as e:
+        click.echo(f"Failed to clone repo: {e}", err=True)
+        exit(4)
+
+
+def update_manifest(file_path, old_path, new_path):
+    """
+    Change the path in a manifest file from old_path to new_path.
+
+    :param file_path: The path to the manifest file.
+    :param old_path: The old path to replace.
+    :param new_path: The new path to use.
+    """
+    click.echo(f"Updating {file_path} to use {new_path} instead of {old_path}")
+    try:
+        with open(file_path, "r+") as file:
+            content = file.read()
+            content = content.replace(old_path, new_path)
+            file.seek(0)
+            file.write(content)
+            file.truncate()
+    except (OSError, IndexError, ValueError) as e:
+        click.echo(f"Failed to update operator.yaml: {e}", err=True)
+        exit(4)
+
+
+@click.command()
+@click.option("--rook-version", default="v1.11.3")
+def main(rook_version: str):
+    rook_repo_url = "https://github.com/rook/rook"
+    destination_dir = "/tmp/rook"
```

Comment:
Please use some tmp directory inside the SNAP_DATA and remove it afterwards. `/tmp/` should not be available in strict confinement.

Reply:
This directory is still used to use the scripts to get the external cluster. Should we still delete it afterwards and ask the user to clone the repo locally and do the steps?